feature-auth/src/main/java/com/example/feature_auth/fragments/login/LoginFragment.kt

```diff
@@ -28,25 +31,31 @@ class LoginFragment : Fragment() {
 
     private val viewModel: LoginViewModel by viewModels()
 
+    private val googleLoginResult =
+        registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+            val task = viewModel.getSignedInAccountFromIntent(result)
+            runCatching {
+                val account = task.getResult(ApiException::class.java)
+                viewModel.signInGoogle(account.idToken, onSuccess = {
+                    requireContext().navigateToActivity(MAIN_ACTIVITY_PATH)
+                })
+
+            }.onFailure {
+                Log.e(tag, it.stackTraceToString())
+            }
+        }
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
         _binding = DataBindingUtil.inflate(inflater, R.layout.fragment_login, container, false)
+        binding.setVariable(BR.viewModel, viewModel)
```

Comment:
You can use apply method to set this variable, for example: 
`DataBindingUtil.inflate(...).apply {
 viewModel = this@LoginFragment.viewModel
}
`